refclient/refclient.go

```diff
@@ -0,0 +1,66 @@
+package refclient
+
+import (
+	"io"
+
+	pb "github.com/buoyantio/strest-grpc/protos"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+func Run() {
+	address := "localhost:11111"
+
+	log.Infof("connecting to %s", address)
+
+	connOpts := []grpc.DialOption{grpc.WithInsecure()}
+	conn, err := grpc.Dial(address, connOpts...)
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	client := pb.NewResponderClient(conn)
+
+	stream, err := client.StreamingGet(context.Background())
+	if err != nil {
+		log.Fatalf("%v.StreamingGet(_) = _, %v", client, err)
+	}
+
+	count := -1
```

Comment:
This invalid input is designed to trigger failures on the server?  In other words, this reference client is more like a unit test, testing a boundry case?

Reply:
That's correct, I just wanted the server to return immediately. A more complete implementation could take the count as a command-line param.

Reply:
Yup, my first instinct was to use a separate repo, but it’s quite convenient having the protobuf already in place for this repo.